### DISCUSSION TITLE
Placeholder doesn't work for Textarea via ->withMeta(...) attributes on page "Create new"

### DISCUSSION THREAD

**Discussion by proart** [laravel/nova-issues#6443] [Ideas & Feature Requests]
- Laravel Version: 8.83.27
- Nova Version: 4.33.2
- PHP Version: 8.1

### Description:
Extra class test__textarea__class works
Placeholder doesn't work:
```php
Textarea::make('Test 123')
    ->withMeta([
        'extraAttributes' => [
            'placeholder' => 'David Hemphill',
            'class' => 'test__textarea__class'
        ]
    ]),
```
Results:
![image](https://github.com/laravel/nova-issues/assets/1822794/cf3ed020-3e27-48a9-8138-e19d3d41468b)
![image](https://github.com/laravel/nova-issues/assets/1822794/d7883a14-7cf5-4673-b113-c4ba2f4e12cc)

Documentation:
![image](https://github.com/laravel/nova-issues/assets/1822794/f12de7b4-9033-4711-95b8-66f8be3b145f)

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0 documentation

We no longer suggest and recommend the following approach. use `placeholder()` method to register placeholder
